models/migrations/v1_22/v282.go

```diff
@@ -0,0 +1,25 @@
+// Copyright 2023 The Gitea Authors. All rights reserved.
+// SPDX-License-Identifier: MIT
+
+package v1_22 //nolint
+
+import (
+	"code.gitea.io/gitea/models/migrations/base"
+
+	"xorm.io/xorm"
+)
+
+func DropDeletedCol(x *xorm.Engine) error {
+	sess := x.NewSession()
+	defer sess.Close()
+	if err := sess.Begin(); err != nil {
+		return err
+	}
+	if err := base.DropTableColumns(sess, "action_runner_token", "deleted"); err != nil {
```

Comment:
Then all deleted records will become undeleted.

Reply:
For action_runner that's true. Sorry for missing that.
For action_runner_token, we did not delete them. But use the `is_active` field to activate/deactivate them.